src/main/java/ua/cv/tim/service/impl/UserServiceImpl.java

```diff
@@ -160,18 +172,22 @@ public void addUser(UserDTO member) throws MessagingException {
 		player.setAlliance(allianceDao.getAllianceByName(member.getAlliance()));
 		user.setPlayer(player);
 		playerDao.add(player);
+		logger.info("User with login {} and id {} was added",user.getLogin(),user.getUuid());
 		String message = "Your login is \"" + user.getLogin() + "\", and password: \""
-				+ password + "\",  role: " + user.getRoles();
-		sendEmail(user, message);
+				+ user.getPassword() + "\",  Role: " + user.getRoles();
```

Comment:
Don't use user.getPassword(). It returns encode password.

you need use variable "password"